website/src/styles/_performance.scss

```diff
@@ -64,6 +69,7 @@
 .perf-out {
     background: var(--sl-color-gray-6);
     margin: 0 !important;
+    overflow: scroll;
```

Comment:
I noticed you added this, is it the correct approach? Shouldn't we reduce the width of the bar on mobile?

Reply:
Yeah that would be nice